index.html

```diff
@@ -1 +1,13 @@
-Aboubacar Kaba
+Aboubacar Kaba
+"About"
+Hi I'm Aboubacar! I'm a web developer who loves problem-solving. After previous career as a financial auditor in a big4 and an oil and gas company, I decided to completely change my professional life and to become a programmer.
+I'm currently completing the intro to programming certificate at Code The Dream.
+"Experience"

```

Comment:
Not sure if the assignment said to use HTML tags (might be next lesson?) but you should definitely try to get semantic in this file!

use a couple `h1`,`h2` tags for headers
`<p>` for your paragraphs
and `div`, `span`, `section` to help group content

Reach out if you have questions!